tests/config_an_test.py

```diff
@@ -62,6 +62,8 @@ def test_config_type(self, ctx):
         result = self.basic_check("type", ["Ethernet0", "Invalid"], ctx, operator.ne)
         assert 'Invalid interface type specified' in result.output
         assert 'Valid interface types:' in result.output
+        result = self.basic_check("type", ["Ethernet16", "Invalid"], ctx, operator.ne)
```

Comment:
maybe i am missing something but how this covers the rj45? what does it mean invalid? should it be yes/know if rj45 is set?

Reply:
It is to test the case that a user inputs an invalid port type, like `Invalid`.
For other port types, it will print all the valid interface types.
For RJ45 port, it prints the error message indicating to set type for RJ45 port is not supported.

`Ethernet16` is an RJ45 port. This is defined in mockdb in PR #4 ([here](https://github.com/keboliu/sonic-utilities/pull/4/files#diff-664d9f6ebb9867be3490d1c8b01caafe38494e14b30c6277aa59b2d86055dff0R137)). As there are 3 PRs for RJ45, all they share the same mock db files, we can not define them in all PRs.
We will add dependency for it.